docs/Manpage.md

```diff
@@ -541,12 +548,46 @@ reinstalled formulae or, every 30 days, for all formulae.
   Install from a bottle if it exists for the current or newest version of macOS, even if it would not normally be used for installation.
 * `--keep-tmp`:
   Retain the temporary files created during installation.
-* `-f`, `--force`:
-  Install without checking for previously installed keg-only or non-migrated versions.
-* `-v`, `--verbose`:
-  Print the verification and postinstall steps.
 * `--display-times`:
   Print install times for each formula at the end of the run.
+* `--cask`:
+  Treat all named arguments as casks.
+* `--appdir`:
```

Comment:
Again, This really bloats the list of options here and I can't see why all of these need specified three times.